Description: DBD::SQLite3 ships its own copy of SQLite. That makes Debian
 sad and so we patched it away.
Origin: vendor
Forwarded: not-neede
Author: Maximilian Gass
Reviewed-by: gregor herrmann <gregoa@debian.org>
Last-Update: 2011-02-12

--- a/Makefile.PL
+++ b/Makefile.PL
@@ -123,7 +123,7 @@
 # a system sqlite is also sophisticated enough to have a patching system
 # that can change the if ( 0 ) to if ( 1 )
 my ($sqlite_local, $sqlite_base, $sqlite_lib, $sqlite_inc);
-if ( 0 ) {
+if ( 1 ) {
 	require File::Spec;
 	if ( $sqlite_base = (grep(/SQLITE_LOCATION=.*/, @ARGV))[0] ) {
 		$sqlite_base =~ /=(.*)/;
@@ -172,11 +172,7 @@
 			}
 		}
 		unless ( $version && ($version >= 3006000) ) {
-			warn "SQLite version must be at least 3.6.0. No header file at that\n";
-			warn "version or higher was found. Using the local version instead.\n";
-			$sqlite_local = 1;
-			undef $sqlite_lib;
-			undef $sqlite_inc;
+                  die "SQLite version must be at least 3.6.0. No header file at that\nversion or higher was found. Using the local version instead.\n";
 		} else {
 			print "Looks good\n" if $ENV{AUTOMATED_TESTING};
 		}
